README.md

```diff
@@ -0,0 +1,3 @@
+    
+    ./setup.sh
+    ./save-0.3.9-linuxX64.kexe . --log all --report-type json --result-output file
```

Comment:
are you sure, that we will need libAnalyzer.so binary in this repo?

Reply:
decided to left as is for now